### PR TITLE
feat(editor): Improve errors in output panel

### PR DESCRIPTION
**Pull request by gandreini**
This PR improves how errors are displayed in the output panel
image

## Related tickets and issues
https://linear.app/n8n/issue/NODE-856/improve-errors-in-output-panel

## Review / Merge checklist
- [x] PR title and summary are descriptive
- [ ] Tests included

### CODE REVIEW

File: packages/workflow/test/errors/node.error.test.ts [hunk 1]
```diff
@@ -3,11 +3,11 @@ import type { INode } from '@/Interfaces';
 import { NodeApiError } from '@/errors/node-api.error';
 import { NodeOperationError } from '@/errors/node-operation.error';
 
-describe('NodeError', () => {
+describe('NodeError TESTER', () => {
```
Comment (alexgrozav) [inline packages/workflow/test/errors/node.error.test.ts:6]: Is this intended?

**Review by alexgrozav — CHANGES_REQUESTED** [review]
Looks and works great overall! Left one small remark.

File: packages/workflow/test/errors/node.error.test.ts (hunk 1 above)
Comment (michael-radency) [reply packages/workflow/test/errors/node.error.test.ts]: nope, removed

**Review by alexgrozav — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.33.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.33.0)

**Pull request merged**